config/main.py

```diff
@@ -7031,5 +7032,67 @@ def del_subinterface(ctx, subinterface_name):
     except JsonPatchConflict as e:
         ctx.fail("{} is invalid vlan subinterface. Error: {}".format(subinterface_name, e))
 
+
+#
+# 'clock' group ('config clock ...')
+#
+@config.group()
+def clock():
+    """Configuring system clock"""
+    pass
+
+
+def get_tzs(ctx, args, incomplete):
+    ret = clicommon.run_command('timedatectl list-timezones',
+                                display_cmd=False, ignore_error=False,
+                                return_cmd=True)
+    if len(ret) == 0:
+        return []
+
+    lst = ret[0].split('\n')
+    lst.append('Etc/UTC')
```

Comment:
Why adding this without check if it is already in the list?

Reply:
`timedatectl list-timezones` don't have it inside, so no need to check.

Reply:
If it is not in the supported list, then how can any set this time zone?

Reply:
Can you grep the output? I tried to grep and I can see Etc/UTC in the output.